### PR TITLE
mate: update to 1.10.0

### PR DESCRIPTION
**Pull request by derlaft**
MATE 1.10.0 is not yet released, but already available at official ftp.

* Removed mate-dialogs
* New library: libmatemixer
* Updated packages
* Non-updated packaged left to be 1.8*

### CODE REVIEW

**Comment by xtraeme** [comment]
Remove the `_release` var and use `${version%.*}` instead.

File: srcpkgs/caja-extensions-common/template [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 pkgname=caja-extensions-common
-version=1.8.0
+version=1.10.0
+_release="${version%.[0-9]*}"
 revision=1
 build_style=gnu-configure
 configure_args="--with-gtk=2.0"
 hostmakedepends="pkg-config intltool"
-makedepends="libcaja-devel mate-desktop-devel dbus-glib-devel gupnp-devel"
+makedepends="libcaja-devel mate-desktop-devel dbus-glib-devel gupnp-devel glib-devel"
```
Comment (xtraeme) [inline srcpkgs/caja-extensions-common/template]: `glib-devel` must be in `hostmakedepends` instead.

File: srcpkgs/libmatemixer/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+distfiles="http://pub.mate-desktop.org/releases/${_release}/${pkgname}-${version}.tar.xz"
+checksum=a4edc285fa07e6f556a3d6e384c7535f26d3a0bbe1cdc56eadd4f0c034ee5551
+
+pre_configure() {
+        autoreconf -fi
+}
+
+libmatemixer-devel_package() {
+    short_desc+=" - development files"
+    depends="alsa-lib-devel pulseaudio-devel ${sourcepkg}>=${version}_${revision}"
```
Comment (xtraeme) [inline srcpkgs/libmatemixer/template]: `depends="${makedepends} ${sourcepkg}>=${version}_${revision}"` instead.

File: srcpkgs/mate-common/template [hunk 3]
```diff
@@ -1,7 +1,8 @@
 # Template file for 'mate-common'
 pkgname=mate-common
 version=1.10.0
-revision=2
+revision=1
+_release="${version%.[0-9]*}"
```
Comment (xtraeme) [inline srcpkgs/mate-common/template]: This change is completely wrong, drop it.

File: srcpkgs/mate-system-monitor/template [hunk 4]
```diff
@@ -2,6 +2,7 @@
 pkgname=mate-system-monitor
 version=1.10.0
 revision=1
+_release="${version%.[0-9]*}"
```
Comment (xtraeme) [inline srcpkgs/mate-system-monitor/template]: This change is also completely wrong, drop it.

It is ok as is.

File: srcpkgs/eclipse/template [hunk 5]
```diff
@@ -10,7 +10,6 @@ license="EPL"
 homepage="http://eclipse.org"
 only_for_archs="i686 x86_64"
 
-_release=luna-SR2
```
Comment (xtraeme) [inline srcpkgs/eclipse/template]: This change is unrelated, drop it.

**Comment by derlaft** [comment]
1) Is rebasing after changes in pull request OK? github does not like, as I can see
2) I have fixed this, but not yet tested building

File: srcpkgs/mate-system-monitor/template [hunk 6]
```diff
@@ -12,5 +12,5 @@ short_desc="A system monitor for MATE"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="LGPL-2"
 homepage="http://mate-desktop.org"
-distfiles="http://pub.mate-desktop.org/releases/${version%.*}/${pkgname}-${version}.tar.xz"
+distfiles="http://pub.mate-desktop.org/releases/${_release}/${pkgname}-${version}.tar.xz"
```
Comment (xtraeme) [inline srcpkgs/mate-system-monitor/template]: This change is unnecessary, drop it.

File: srcpkgs/mate-terminal/template [hunk 7]
```diff
@@ -1,15 +1,15 @@
 # Template file for 'mate-terminal'
 pkgname=mate-terminal
-version=1.8.1
+version=1.10.0
 revision=1
 build_style=gnu-configure
 configure_args="--with-gtk=2.0"
-hostmakedepends="pkg-config intltool itstool glib-devel"
+hostmakedepends="pkg-config intltool itstool glib-devel mate-desktop-devel"
```
Comment (xtraeme) [inline srcpkgs/mate-terminal/template]: `mate-desktop-devel` must be in `makedepends` instead.

**Comment by xtraeme** [comment]
1- Rebasing is ok
2- We'll see if travis is able to build it

File: srcpkgs/eom/template [hunk 8]
```diff
@@ -1,20 +1,20 @@
...
+version=1.10.0
 revision=1
 build_style=gnu-configure
 configure_args="--with-gtk=2.0 --disable-python"
-hostmakedepends="pkg-config intltool itstool"
+hostmakedepends="pkg-config intltool itstool mate-common"
 makedepends="gtk+-devel dbus-glib-devel exempi-devel lcms2-devel libexif-devel
  libjpeg-turbo-devel librsvg-devel mate-desktop-devel startup-notification-devel
- mate-icon-theme"
+ mate-icon-theme $(vopt_if gir gobject-introspection)"
```
Comment (xtraeme) [inline srcpkgs/eom/template]: `vopt_xxx` helpers are useless if you don't see `build_options` and `build_options_default`.

See `srcpkgs/atk` as example.

File: srcpkgs/mate-desktop/template [hunk 9]
```diff
@@ -1,19 +1,22 @@
...
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="LGPL-3"
 homepage="http://mate-desktop.org"
-distfiles="http://pub.mate-desktop.org/releases/1.8/${pkgname}-${version}.tar.xz"
-checksum=0ed978e40b12d12cee2b9eb2457a9c9d43a15224ba18271494b1fde5f5bcf3cb
+distfiles="http://pub.mate-desktop.org/releases/${version%.*}/${pkgname}-${version}.tar.xz"
+checksum=153b9636314b6271084320c69925f877c6a7b51f498bcb34e86eca962baa0b61
+
+# Package build options
+build_options="gir"
```
Comment (xtraeme) [inline srcpkgs/mate-desktop/template]: You are missing here:

```
if [ -z "$CROSS_BUILD" ]; then
   build_options_default="gir"
fi
```

File: srcpkgs/mate-desktop/template [hunk 10]
```diff
@@ -1,19 +1,22 @@
 # Template file for 'mate-desktop'
 pkgname=mate-desktop
-version=1.8.2
+version=1.10.0
 revision=1
 build_style=gnu-configure
 configure_args="--with-gtk=2.0 --disable-static --enable-mpaste
- --disable-schemas-compile --disable-desktop-docs"
-hostmakedepends="pkg-config intltool itstool glib-devel"
+ --disable-schemas-compile $(vopt_enable gir gobject-introspection)"
```
Comment (xtraeme) [inline srcpkgs/mate-desktop/template]: This is probably wrong, the `configure` option is probably called `--enable-introspection` not `--enable-gobject-introspection`.

File: srcpkgs/atril/template [hunk 11]
```diff
@@ -1,13 +1,13 @@
...
-revision=2
+version=1.10.0
+revision=1
 build_style=gnu-configure
 build_options="gir"
 configure_args="--with-gtk=2.0 --disable-schemas-compile --enable-djvu
  --enable-dvi --enable-t1lib --enable-pixbuf --enable-comics --enable-xps
  --disable-static $(vopt_enable gir introspection)"
-hostmakedepends="pkg-config intltool itstool glib-devel"
+hostmakedepends="pkg-config intltool itstool glib-devel "$(vopt_if gir introspection)
```
Comment (xtraeme) [inline srcpkgs/atril/template]: `$(vopt_if gir gobject-introspection)`

**Comment by derlaft** [comment]
I have noticed that eom build fails only if proot is used.

File: srcpkgs/atril/template (hunk 11 above)
Comment (xtraeme) [inline srcpkgs/atril/template]: The `"` should be after the `$(vopt_if)`
